INeedCoffee/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
.idea files are for RubyMine or related IDEs. These are not needed in the repo. Add them to .gitignore.